content/5_Plat/VTree.problems.json

```diff
@@ -0,0 +1,67 @@
+{
+  "MODULE_ID": "VT",
+  "uniqueocc": [
+    {
+      "uniqueId": "cf-1681F",
+      "name": "Unique Occurrences",
+      "url": "https://codeforces.com/contest/1681/problem/F",
+      "source": "CF",
+      "difficulty": "Easy",
+      "isStarred": true,
+      "tags": ["Virtual Tree"],
+      "solutionMetadata": {
+        "kind": "none"
+      }
+    }
+  ],
+  "general": [
+    {
+      "uniqueId": "atcoder-abc340_g",
+      "name": "Leaf Color",
+      "url": "https://atcoder.jp/contests/abc340/tasks/abc340_g",
+      "source": "AC",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Virtual Tree"],
+      "solutionMetadata": {
+        "kind": "none"
+      }
+    },
+    {
+      "uniqueId": "cf-613D",
+      "name": "Kingdom and its Cities",
+      "url": "https://codeforces.com/contest/613/problem/D/",
+      "source": "CF",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Virtual Tree"],
+      "solutionMetadata": {
+        "kind": "none"
+      }
+    },
+    {
+      "uniqueId": "cf-1292D",
+      "name": "Chaotic V.",
+      "url": "https://codeforces.com/contest/1292/problem/D",
+      "source": "CF",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Virtual Tree"],
+      "solutionMetadata": {
+        "kind": "none"
+      }
+    },
+    {
+      "uniqueId": "cfgym-100551D",
+      "name": "Bridges: The Final Battle",
+      "url": "https://codeforces.com/problemset/gymProblem/100551/D",
+      "source": "Gym",
+      "difficulty": "Insane",
+      "isStarred": true,
```

Comment:
why is this starred?

Also, the topics required to solve this problem have not been introduced in prerequisite modules.

Reply:
fine to include it, just don't star it

Reply:
note would be helpful too